eng/pipelines/coreclr/libraries-pgo.yml

```diff
@@ -37,6 +37,7 @@ extends:
                   extraHelixArguments: /maxcpucount:10
                   scenarios:
                     - defaultpgo
+                    - jitrpolayout
```

Comment:
We haven't historically stressed Windows arm64 since we've been short on CI resources, and codegen is similar to Linux arm64. So perhaps leave this part out?

Reply:
Sure thing.